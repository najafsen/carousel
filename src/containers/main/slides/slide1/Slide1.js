import React from "react";
import commaGreen from "./img/comma-green.svg";
import face from "./img/face.svg";
import "./Slide1.scss";

export const Slide1 = () => (
  <div className="slide-content">
    <div className="slide1">
      <img src={commaGreen} alt="Comma Green" />
      <h1>
        This piece of software is <i>the shit</i>,
        <br />
        I’ve never tried anything like it.
      </h1>
      <img className="bottom-image" src={face} alt="face" />
      <h4>Mr. Face</h4>
      <p>Karma Guru @ Face Co™</p>
    </div>
  </div>
);
