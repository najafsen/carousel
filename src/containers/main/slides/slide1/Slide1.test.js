import React from "react";
import { render } from "@testing-library/react";
import { Slide1 } from "./Slide1";

describe("Slide1", () => {
  test("matches snapshot", () => {
    const { asFragment } = render(<Slide1 />);
    expect(asFragment()).toMatchSnapshot();
  });
});
