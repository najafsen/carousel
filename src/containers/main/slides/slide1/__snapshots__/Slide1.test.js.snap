// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slide1 matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="slide-content"
  >
    <div
      class="slide1"
    >
      <img
        alt="Comma Green"
        src="comma-green.svg"
      />
      <h1>
        This piece of software is 
        <i>
          the shit
        </i>
        ,
        <br />
        I’ve never tried anything like it.
      </h1>
      <img
        alt="face"
        class="bottom-image"
        src="face.svg"
      />
      <h4>
        Mr. Face
      </h4>
      <p>
        Karma Guru @ Face Co™
      </p>
    </div>
  </div>
</DocumentFragment>
`;
